scipy/spatial/transform/_rotation.pyx

```diff
@@ -2744,7 +2745,7 @@ cdef class Rotation:
         quat[:, 2] *= -1
         if self._single:
             quat = quat[0]
-        return self.__class__(quat, copy=False)
+        return self.__class__(quat, normalize=False, copy=False)
```

Comment:
Just a note about the current implementation - for some reason the 3 vector components are multiplied by -1 (instead of the single scalar). 

In line with "micro optimization" might consider changing that. But I got an impression that people were quite sensitive to minor "convention" changes, so probably it's better not to change that.

Reply:
That's on purpose, it's an inversion in SU(2) that was done to preserve the double cover property of quaternions for advanced users, see: https://github.com/scipy/scipy/pull/18955

Reply:
`q * [1, 1, 1, -1]` is an inversion in SO(3), but not SU(2). `q * [-1, -1, -1, 1]` is proper inversion in both. See this comment and the broader discussion: https://github.com/scipy/scipy/pull/17334#issuecomment-1324405039

Reply:
OK, I see that it is necessary for such purposes, makes sense.